toolkits/global/packages/global-corporate-footer/README.md

```diff
@@ -1,29 +1,148 @@
+   the above)
+
+For **the first one**, please refer to the [Preference Dialog Trigger section of the
+Cookie consent
+documentation](https://cookie-consent.public.springernature.app/docs/getting-started/installation/#preference-dialog-trigger)
+to get the implementation details for the button.  
+An example implementation is to be found in the [`demo` folder](https://github.com/springernature/frontend-toolkits/tree/master/toolkits/global/packages/global-corporate-footer/demo).
+
+For **the second one**, the generated HTML of the _Manage cookies / Do not sell data_
+interactive element should be:
 
-### HTML
 ```html
-<div class="c-corporate-footer">
-    <div class="u-container">
-        <img src="images/logos/sn-logo-white.svg" alt="Springer Nature" loading="lazy" width="140" height="14"/>
-        <p class="c-corporate-footer__legal" data-test="copyright">&copy; 2020 Springer Nature Limited</p>
-    </div>
-</div>
+<button onclick="Optanon.ToggleInfoDisplay()" class="c-corporate-footer__link">Manage cookies / Do not sell my data</button>
+```
+
+## Footer with extended links
+
+Links that **can be present** depending on the product needs:
+- Terms
+- California Privacy Statement
+- Help & Support
+
+**Important**: The label for the _Terms_ link is not yet standardized, as
+opposed to the two other ones. Hence _Terms_ may consist of more than 1 item.  
+The destination of the link may differ on a product basis for _Terms_ and _Help
+& Support_.  
+However the destination of the link for _California Privacy Statement_ should be
+fixed to the one to be found in the  [data source of the
+Demo](https://github.com/springernature/frontend-toolkits/tree/master/toolkits/global/packages/global-corporate-footer/demo/context.json).
+
+**Note**: _California Privacy Statement_ and _Help & Support_ may be omitted on
+a per project basis.
+
+## Template
+
+Find a configurable template in the [`view` folder](https://github.com/springernature/frontend-toolkits/tree/master/toolkits/global/packages/global-corporate-footer/view). 
+
+You can see an example in the [`demo` folder](https://github.com/springernature/frontend-toolkits/tree/master/toolkits/global/packages/global-corporate-footer/demo).
+
+### Template configuration
+
+The configuration (a.k.a context) of the `corporate-footer` template should have
+the following structure:
+
+```json
+{
+  "navigation": {
+  "links": [
+    {
+      "text": "Example link",
+      "url": "/url/to/example/link"
+    },
+    {
+      "text": "Example button",
+      "buttonProperties": "onlick=\"doSomething()\""
+    }
+  ]
+  },
+  "image": {
+    "src": "path/to/springer/nature/logo",
+    "alt": "Springer Nature",
+    "link": "https://www.springernature.com/"
+  },
+  "currentYear": 2022
+}
 ```
 
-Note that a `<footer>` region/landmark is not included. This allows you to combine this component with—for example—`springer-publisher-footer` inside a common `<footer>`.
```

Comment:
@morgaan Hi! What's the reasoning behind removing this line? I added this guidance to aid in creating an accessible page structure.

Reply:
Oh great catch Heydon thank you!
That was not intentional, I most likely missed it when revamping the all README.md. Thank god we have reviews :).
I will put it back as soon as @amyhupe is done with the changes to the README, and be sure before merging

Reply:
@morgaan Ah brilliant, thanks!